.github/workflows/orca.yml

```
@@ -92,6 +92,31 @@ jobs:
           php-version: ${{ matrix.php-version }}
           coverage: xdebug
 
+      - name: Install google-chrome-stable
+        run: |
+          # Remove existing google chrome and install required version.
+          sudo dpkg -r google-chrome-stable
+          CHROME_VERSION_STRING="114.0.5735.198-1"
+          wget --no-verbose -O /tmp/chrome.deb https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_114.0.5735.198-1_amd64.deb \
+          && sudo apt install -y /tmp/chrome.deb \
+          && sudo rm /tmp/chrome.deb
+          sudo echo "CHROME_BIN=/usr/bin/google-chrome" | sudo tee -a /etc/environment
+
+          # Download and unpack chromedriver.
+          CHROME_DRIVER_VERSION_STRING="114.0.5735.90"
+          CHROMEDRIVER_URL="https://chromedriver.storage.googleapis.com/${CHROME_DRIVER_VERSION_STRING}/chromedriver_linux64.zip"
+          CHROMEDRIVER_ARCHIVE="chromedriver_linux64.zip"
```

I believe you can replace another hardcoded value with your variable again here: (from `"chromedriver_linux64.zip"` to `${CHROMEDRIVER_ARCHIVE}.zip`)
```suggestion
          CHROMEDRIVER_ARCHIVE="chromedriver_linux64.zip"
          CHROMEDRIVER_URL="https://chromedriver.storage.googleapis.com/${CHROME_DRIVER_VERSION_STRING}/${CHROMEDRIVER_ARCHIVE}.zip"
```